bigquery/tests/unit/test_table.py

```diff
@@ -1174,7 +1170,7 @@ def test_ctor(self):
         self.assertEqual(table.time_partitioning.field, "mycolumn")
         self.assertEqual(table.labels["some-stuff"], "this-is-a-label")
         self.assertIsNone(table.view_use_legacy_sql)
-        self.assertIsNone(table.expires)
+        # self.assertIsNone(table.expires)
```

Comment:
Remove, don't comment out.

Actually, could you move this `assertIsNone` for `expires` and add one for  `created` to `test_ctor_missing_properties`?

Reply:
Thanks for removing. Could you add assertions for `None` values to the `test_ctor_missing_properties` test, too?

Reply:
Added assertions for created and expires under the `test_ctor_missing_properties` test.